tests/dragonfly/connection_test.py

```diff
@@ -1159,6 +1159,32 @@ async def push_pipeline(bad_actor_client, size=1):
     assert info["dispatch_queue_bytes"] == 0
 
 
+@dfly_args({"proactor_threads": 4, "pipeline_queue_limit": 100})
+async def test_pipeline_overlimit(df_factory: DflyInstanceFactory):
```

Comment:
This crashes without #4789 right ?

Reply:
crashes on my laptop, I am trying to see if I see crash in github also

Reply:
Ok it does not fail on github, but it fails constantly on my laptop.
I believe this is because the slowness of the github machine.
I suggest to just add this test as it